string block_id
geometry_msgs/PoseStamped pose